js/script.js

```diff
@@ -0,0 +1,56 @@
+import { $taskInput, $taskContainer, $addButton, $errorMessage} from "./element.js";
+import { sanitizeInput } from "./utility.js";
+
+let taskList = [];
+
+const addTaskHandler = () => {
+    const taskTitle = sanitizeInput($taskInput.value);
+    
+    if (!taskTitle.length) {
+        $errorMessage.hidden = false;
+        $errorMessage.innerHTML = "Task Name must be provided";
+        return;
+    }
+    else{
+        $errorMessage.hidden = true;
+    }
+
+    const task = createTask(taskTitle);
+
+    taskList.unshift(task);
+
+    const $taskElement = createTaskElement(task);
+    $taskContainer.prepend($taskElement);
+};
+
+const createTask = (taskTitle) => {
+    return {
+        "id": Date.now(),
+        "title": taskTitle,
+        "createdAt": new Date().toLocaleDateString()
+    }
+};
+
+const deleteTaskHandler = (event) => {
+    const $taskElement = event.target.parentElement;
+    const taskId = $taskElement.id;
+
+    $taskElement.remove();
+    taskList = taskList.filter((task) => task.id!=taskId);
```

Comment:
`task.id!==taskId`
also please reformate the code

Reply:
@rka-wtag bhai task.id!==taskId will not work because taskId is retrieved from the DOM element so it is a string where as task.id is a number. So here we need to do loose equality for the check. Another way is we can convert the string to number and then do strict equality.

Reply:
@aks-wtag bhai, I think convert the string to number is the better approach.

Reply:
@rka-wtag and @ich-wtag bhai thanks for the suggestion